### DISCUSSION TITLE
756条数据，无底模3K步，loss在100左右震荡还有必要跑下去吗？

### DISCUSSION THREAD

**Discussion by fatinghenji** [svc-develop-team/so-vits-svc#110] [Q&A]
RT，补充loss图片：
![image](https://user-images.githubusercontent.com/37131177/229169218-891c6d5f-5327-4187-80ec-eaf79dd5b70e.png)

**Comment by fatinghenji** [comment]
在考虑加个底模进去重新跑，但是不知道4.0v2的底模哪里有（

**Reply by fatinghenji** [reply]
划掉，好像还能救救，跑到5k步现在loss到90+了。
![image](https://user-images.githubusercontent.com/37131177/229173896-1e37f2cf-e241-41d0-9ad3-c859791db3b2.png)

**Comment by JoshuaViolet** [comment]
我是笨比 我可以问一下这个图怎么查看吗

**Reply by ZhouhaoJiang** [reply]
我也是笨比 我可以问一下这个图怎么查看吗
